apps/docs/content/guides/functions/debugging.mdx

```diff
@@ -5,11 +5,32 @@ description: 'Debugging tips and Edge Function limitations.'
 subtitle: 'Debugging tips and Edge Function limitations.'
 ---
 
-## Logs & debugging
+## Debugging
```

Comment:
@thorwebdev I think it's good to break this into a separate page so it's easier to find. Page title can be Local Debugging or Debugging tools (the latter is what Cloudflare uses https://developers.cloudflare.com/workers/testing/debugging-tools/)